lib/openstudio-workflow.rb

```diff
@@ -153,3 +153,73 @@ def load_adapter(name, adapter_options = {})
     end
   end
 end
+
+# Extend OS Runner to persist measure information throughout the workflow
+class ExtendedRunner < OpenStudio::Ruleset::OSRunner
+  # Allow former arguments to be set and read
+  #todo: Consider having the former arguments passed in in initialization, and define as attr_reader
+  attr_accessor :former_workflow_arguments
```

Comment:
That was the same problem I hit in OS, the only solution would be to pass old results into the new runner somehow, probably much easier to do this in WorkflowGem than in RunManager.  Let me know if you have any questions, sounds like you have a handle on it though.